test/localtest.sh

```diff
@@ -79,4 +79,28 @@ if [ -n "${EXPECTED_TOOL_VERSION}" ]; then
     fi
 fi
 
+system_probe_config_file=/etc/datadog-agent/system-probe.yaml
+if [ -n "${DD_SYSTEM_PROBE_ENABLED}" ]; then
+    if [ -e "$system_probe_config_file" ]; then
+        echo "[OK] Found system-probe configuration file $system_probe_config_file"
```

Comment:
use here shell colors as well, green for OK, red for FAIL

Reply:
that's not being used in the entire script, I don't think it is in the scope of the PR